docs/theia.md

```diff
@@ -0,0 +1,68 @@
+# Theia
```

Comment:
We should use "theia" to refer to the CLI tool.

Maybe use "theia Command-line Tool" or "Theia Command-line Tool" as the title

Reply:
Thanks, changed to "Theia Command-line Tool" now.